generate_models.py

```diff
@@ -1468,23 +1506,43 @@ def generate_urbs_model(paths, param):
     # create empty dictionary
     urbs_model = {}
     # read .csv files and associate them with relevant sheet
-    for str in urbs_paths:
+    for name in urbs_paths:
```

Comment:
it is funny how you replaced "str" with "name", which is maybe a reserved word as well?
I could not check that, but I am suspicious and I would avoid that

Reply:
name is not a reserved world, str was though. I replaced name by filename just to be on the safe side